ansible/roles/touchforms/templates/localsettings.py.j2

```diff
@@ -33,3 +33,5 @@ POSTGRES_DATABASE = {
         'PREPARE_THRESHOLD': 0
 }
 ### END POSTGRES CONFIG ###
+
+FORMPLAYER_INTERNAL_AUTH_KEY={{ localsettings.FORMPLAYER_INTERNAL_AUTH_KEY }}
```

Comment:
this value should have quotes around it, touchforms wouldn't start after today's deploy because it's missing them

i fixed it in prod's touchforms localsettings (the only one in use right now) but this should be fixed and updated in all envs this was deployed to